fantasybot/api.py

```
@@ -116,6 +116,25 @@ def lineup(self, team_id):
+        if not fetch_all:
+            return self.get(self._cmp(f"/leagues/{league_id}/activity/0?x-lang=es"))
+        all_acts = []
+        idx = 0
+        while True:
+            try:
+                r = self.get(self._cmp(f"/leagues/{league_id}/activity/{idx}?x-lang=es"))
+                if not r:
+                    break
+                all_acts.extend(r)
+                idx += 1
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)